dokka-integration-tests/maven/src/main/kotlin/settingsXml.kt

```diff
@@ -12,7 +12,7 @@ import java.nio.file.Paths
 
 /** Create `settings.xml` file contents, with the custom dev Maven repos. */
 @Language("xml")
-fun createSettingsXml(): String {
+public fun createSettingsXml(): String {
```

Comment:
Probably it makes sense to disable explicit mode in the integration tests convention plugin